containers/payments/Method.js

```diff
@@ -13,9 +13,11 @@ import Bitcoin from './Bitcoin';
 const { CARD, PAYPAL, BITCOIN, CASH } = PAYMENT_METHOD_TYPES;
 
 const Method = ({ type, amount = 0, currency, onCard, onPayPal, method, methods, loading }) => {
-    const { card, updateCard, errors, isValid } = useCard();
+    const cache = useCache();
+    const { card, updateCard, errors, isValid } = useCard(cache.get('card'));
 
     useEffect(() => {
+        cache.set('card', card);
```

Comment:
Could we find another solution than using `cache` for this?

Reply:
I guess we could cache card data in a variable outside the component, but I'm not sure if this PR is even relevant to the project for the reasons @vincaslt mentioned.